node/node_newblock.go

```diff
@@ -80,11 +80,6 @@ func (node *Node) WaitForConsensusReadyv2(readySignal chan struct{}, stopChan ch
 					viewID := node.Consensus.GetViewID()
 					// add aggregated commit signatures from last block, except for the first two blocks
 
-					err = node.Worker.UpdateCurrent(coinbase)
```

Comment:
who made this change?  what's the reason of making this change here?

Reply:
RJ made. i synced with him today.

Reply:
Somehow if i remove the commitTransactions - tx is not working anymore